File: examples/modulith-clinical-tool-jpa/src/main/java/io/zenwave360/example/clinicaltool/modules/surveys/core/inbound/dtos/SurveysByHospital.java
package io.zenwave360.example.clinicaltool.modules.surveys.core.inbound.dtos;

import io.zenwave360.example.clinicaltool.modules.surveys.core.domain.*;
import java.io.Serializable;
import java.math.*;
import java.time.*;
import java.util.*;
import jakarta.validation.constraints.*;

/**
 * SurveysByHospital.
 */
@lombok.Getter @lombok.Setter
public  class SurveysByHospital  implements Serializable {


    @NotNull
    private Long hospitalId ;

    @Size(max = 3)
    private String lang ;





}

File: examples/modulith-klinical-tool-jpa/src/main/kotlin/io/example/asyncapi/provider/clinicaltool/modules/surveys/service/dtos/SurveysByHospital.kt
package io.example.asyncapi.provider.clinicaltool.modules.surveys.service.dtos

import io.zenwave360.example.clinicaltool.modules.surveys.domain.*
import java.io.Serializable
import java.math.*
import java.time.*
import jakarta.validation.constraints.*

/**
 * SurveysByHospital.
 */
data class SurveysByHospital(

    @NotNull
    val hospitalId: Long?  = null,

    @Size(max = 3)
    val lang: String?  = null

)  : Serializable {




}
